crates/bevy_render/src/entity.rs

```diff
@@ -77,3 +86,40 @@ impl Default for Camera2dBundle {
         }
     }
 }
+
+/// A component bundle for "orthographic camera" entities
+///
+/// Uses a normalized orthographic projection.
+///
+/// Use this for CAD-like 3D views or non-pixel-oriented 2D games.
```

Comment:
A good example of a game which uses this kind of perspective is [Into the Breach](https://subsetgames.com/itb.html)


![](https://subsetgames.com/img/itb/ITB_main_animation2.gif)

Reply:
That is, `2.5d` is normally the term used to describe this style of projection for 3d scenes

Reply:
I don't think that's a good example. A game with a pixel-art aesthetic like that is probably better done with a pixel-sized 2D orthographic projection (like the default in bevy), with 2D tiles/sprites with suitable positioning and draw order. Otherwise, good luck getting your pixel art to align and look pretty.

A better example is something like School Tycoon:

![](https://user-images.githubusercontent.com/40234599/105541990-ea734600-5cf8-11eb-938c-ebe2571f7326.png)

This game actually uses 3D models with a freely-rotating overhead camera, but with an orthographic projection to give that "isometric" look.

Reply:
I suppose captain toad treasure tracker is another example, which I'm fairly certain actually is 3d